charts/istio-authorizer/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: istio-authorizer
 description: A Helm chart for istio-authorizer
 type: application
-version: 2.4.2-1
-appVersion: "2.4.1"
+version: 2.5.0
+appVersion: "2.5.0"
```

Comment:
format

Reply:
Could you point what is not well formated?

Reply:
no empty line at the end